src/main/java/com/flightstats/hub/app/HubBindings.java

```diff
@@ -177,6 +212,12 @@ protected void configure() {
         bind(InFlightService.class).asEagerSingleton();
         bind(ChannelService.class).asEagerSingleton();
 
+
+        bind(HubVersion.class).toInstance(new HubVersion());
```

Comment:
I can't find any reason this wasn't bound in this manner before

Reply:
My understanding of guice was that it auto-wired dependencies if there wasn't any fancy settings required (like  special scoping or having to specify a provider or an implementation of an interface)... Was this throwing a DI error without the bind?

Reply:
It will error now if not bound, as there is no reference to the injector available here at this point in the lifecycle . Before however I think it would have been easier to bind it for testing ease, as well as the fact we are calling injector.getInstance a couple times (through HubProvider) to get it. I don't know the rule of thumb on that but my first thought was that if calling more than once it might be cleaner to bind it. How do you feel about that?

Reply:
That works for me. I think I was having issues wrapping my head around the problem when you explained it before...maybe because of all the statics we have with HubProvider and HubProperties? Thanks.